samples/examples/person-card.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
```

Comment:
This examples doesn't seem necessary since we have a similar one in storybook

Reply:
Good point, will remove